src/components/navigationsLinks.js

```diff
@@ -1,19 +1,21 @@
-let counter = 0;
+import { v4 as uuidv4 } from 'uuid';
 
-const generateUniqueId = () => {
-  counter += 1;
-  return `menu-item-${counter}`;
-};
+// let counter = 0;
+
+// const generateUniqueId = () => {
+//   counter += 1;
+//   return `menu-item-${counter}`;
+// };
```

Comment:
- Great Job for fixing previous reviewers' suggestions. I think it would be better if you Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍